cmake/extensions.cmake

```diff
@@ -1960,6 +1960,47 @@ Relative paths are only allowed with `-D${ARGV1}=<path>`")
   endif()
 endfunction()
 
+# Usage:
+#   zephyr_string(<mode> <out-var> <input> ...)
+#
+# Zephyr string function extension.
+# This function extends the CMake string function by providing additional
+# manipulation arguments to CMake string.
+#
+# SANITIZE: Ensure that the output string does not contain any special
+#           characters. Special characters, such as -, +, =, $, etc. are
+#           converted to an underscores '_'.
```

Comment:
```suggestion
#           converted to underscores '_'.
```
Same below.

Reply:
There's one more such "an" [right below](https://github.com/zephyrproject-rtos/zephyr/pull/31422/files#diff-c4994ce9c473874bf18eb76dc67b5cfc1ecc2171de4d4ddef83ddfdab6bef559R1976).

Reply:
and fixed.